content/guides/kubernetes/contour-to-ingress-and-beyond.md

```diff
@@ -0,0 +1,400 @@
+---
+title:  "Getting Started with Contour - To Ingress and Beyond"
+linkTitle: "Getting Started with Contour - To Ingress and Beyond"
+parent: Contour to Ingress and Beyond
+weight: 2
+topics:
+- Kubernetes
+- Tanzu
+tags:
+- Kubernetes
+- Tanzu
+- Contour
+- Ingress
+- httproxy
+- istio
+- service mesh
+- network
+# Author(s)
+team:
+- Paul Czarkowski
+---
+
+### Introduction to Contour
+
+Need to write some details about Contour here
```

Comment:
FYI, adding an article on Contour as part of this PR: https://github.com/vmware-tanzu/tanzu-dev-portal/pull/516/files - Might be good enough as a "what is" to precede this one and cover some of the intro piece here.

Reply:
yeah I figured we had a few examples that use contour, that we could skip the real basics